observability/tools/Kubernetes-Dashboard.md

```diff
@@ -0,0 +1,38 @@
+# Kubernetes Dashboard
```

Comment:
Echoing @ams0's comment about the k8s dashboard being deprecated. Most (if not all) of the benefits stated below from using the provided k8s dashboard are now available in alternative OSS dashboard projects. Since this playbook is meant for reference in future projects and the timing of the k8s dashboard deprecation, we should refactor this to be a more generic page with regards to UI dashboards and cite/link some of the alternatives.

Reply:
Thank you @khaines - This helps. I will work to refactor this.

Reply:
Updated based on the feedback, please re-review and let me know of any other changes! Appreciate your help!